jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -8521,6 +8524,19 @@ public String translateEscapes() {
 					}
 					builder.append((char)octal);
 					break;
+				/**
+				 * JEP 368: Text Blocks (Second Preview)
+				 * '\r', "\r\n" and '\n' are ignored as per new continuation \<line-terminator> escape sequence 
+				 * i.e. ignore line terminator and continue line
+				 * */
```

Comment:
@fengxue-IS this is the added comment to explain why we're ignoring the line terminators